vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -295,6 +295,12 @@ components:
         iommu:
           type: boolean
           default: false
+        num_queues:
+          type: integer
+          default: 2
+        queue_size:
+          type: integer
+          default: 256
```

Comment:
Thanks for adding this, could you please squash it with the commit `vmm: Add multiple queue option and update config for virtio-net device`.

Reply:
squash done!